rts/at/util/src/main/java/org/jboss/jbossts/star/util/TxSupport.java

```diff
@@ -482,6 +483,8 @@ public String httpRequest(int[] expect, String url, String method, String mediaT
                 body = (status != -1 ? getContent(connection) : "");
             } catch (IOException e) {
                 body = "";
+                RESTATLogger.at18NLogger.warn_rest_at_txSupportHttpRequest(e.getMessage(),e);
```

Comment:
I'm not sure but here it seems the exception is an "expected state". There should not be logged warning as until now the exception was just ignored. I think it's good to not hid exception but I recommend to use `debug` or `trace` level but not warning.

Reply:
+1

Reply:
Changed to DEBUG